test/integration/models/test_linode.py

```diff
@@ -335,7 +358,7 @@ def test_linode_initate_migration(get_client):
 
     wait_for_condition(10, 100, get_status, linode, "running")
     # Says it could take up to ~6 hrs for migration to fully complete
-    linode.initiate_migration(region="us-central")
+    linode.initiate_migration(region="us-central", migration_type="warm")
```

Comment:
```suggestion
    linode.initiate_migration(region="us-central", migration_type="cold")
```
IIRC, warm migration is not available for cross DC migration